src/odemis/gui/comp/viewport.py

```diff
@@ -1516,24 +1494,3 @@ def _on_new_data(self, data):
             self.clear()
 
         self.Refresh()
-
-    def _on_pixel_select(self, pixel):
-        """ Clear the marking line when the selected pixel is cleared """
-        if None in pixel:
-            self.canvas.markline_overlay.clear_labels()
```

Comment:
Need to check whether that's still useful to do. If so, it probably could go into `self.clear()`.

Reply:
Not needed, as it's already done by `TwoDPlotCanvas.clear()`

Reply:
Yes, now, `_on_new_data()` detects that .image is None, and `_connect_stream()` detects that there are no streams to show. In both case, `.clear()` is called, and eventually the overlay is hidden.